pyflask/manageNeuroconv/manage_neuroconv.py

```diff
@@ -69,6 +73,35 @@ def get_metadata_schema(source_data: Dict[str, dict]) -> Dict[str, dict]:
     return json.loads(json.dumps(dict(results=metadata, schema=schema), cls=NWBMetaDataEncoder))
 
 
+class objectview(object):
+    def __init__(self, d):
+        self.__dict__ = d
+
+
+def validate_metadata(parent: dict, function: str) -> dict:
```

Comment:
```suggestion
def validate_metadata(nwbfile_metadata: dict, check_function_name: str) -> dict:
```

Some suggestions for better readability of the code; we nearly always use full words that are as descriptive as possible for Python code

Only exceptions are usually built-in names, which I will point out that `cls` is actually one of those (used as the first argument to class methods)

Reply:
Also, `_name` is more honest to the expected `str` usage; it is not uncommon to see `function: callable` in Python, but that's not how this function signature is setup (nor would I recommend it either, this approach seems fine)